ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -30,6 +30,12 @@ SD:
   - supply_air_heating_flowrate_capacity
   - cooling_thermal_power_capacity
   - supply_air_temperature_sensor
+  - zone_air_occupied_cooling_temperature_setpoint
```

Comment:
These do not belong here. Create an abstract type for this information, it has nothing to do with airflow control, which is the purpose of the type.

Reply:
Updated